contrib/hadoop-store-builder/test/voldemort/store/readonly/fetcher/HdfsFetcherTest.java

```diff
@@ -168,4 +168,291 @@ public void testFetch() throws Exception {
         checkSumFile.delete();
 
     }
+
+    public void testAggStatsWithValidFile() throws Exception {
+        HdfsFetcherAggStats stats = HdfsFetcherAggStats.getStats();
```

Comment:
I believe we are validating all parameters here. Can we validate only the things, that we expect to change.

Or other way is to have a copyConstructor for the HdfsFetcherAggStats and also make the method implement equals. This will reduce the amount of boiler plate code in the test.